governance.md

```diff
@@ -47,15 +47,15 @@ Duties
 
 Qualification guidelines
 
+* Is already fulfilling the Reviewer role
 * Belongs to an organization with significant stake in project
 * Known active participant in the community
 * Has submitted at least 5 PRs of significant scope that are merged
-* Has reviewed 5 PRs of significant scope
 * Sponsored by 2 Approvers
 
 [Current Approvers](https://github.com/spinnaker/spinnaker/blob/master/approvers.md)
 
-If you’d like to nominate yourself as an Approver, please create an [Issue in the community repository](https://github.com/spinnaker/community/issues). 
+If you’d like to nominate yourself as an Approver, please create an [Issue in the community repository](https://github.com/spinnaker/community/issues). If you are already a Reviewer who meets the above Qualifications, you do not need to wait before submitting for Approver status.
```

Comment:
I'm a bit confused by the new language.  So if someone belongs to an organization with significant stake in the project and wants to become an approver, they can:
* Submit an application for reviewer
* Immediately upon approval of that reviewer nomination, submit a new application for approver

I'm not sure I see the benefit in requiring someone to first become a reviewer if we're explicitly allowing them to immediately jump to approver (and dropping the requirement of having performed any reviewer duties).

Reply:
Yeah, it does seem a tad confusing. Really, I think dropping the `Has reviewed 5 PRs of significant scope` back in would clear it up. If anything, it at least give you a bar to hit, since there isn't one defined in the `Reviewer guidelines` section.

Reply:
+1 to having at least *some* additional criterion reviewers need to fulfil before being eligible to become approvers. Or do we genuinely feel there is no such criterion?

Reply:
I agree with Andrew's comment.